<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewBooks</class>
 <widget class="QWidget" name="addNewBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_bookImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>点击上传书籍图片</string>
   </property>
   <property name="iconSize">
    <size>
     <width>161</width>
     <height>191</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bookName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请完整填入书籍名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bookInfo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请填入出版社名称及版本信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bookPrice">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请填写意向价格</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bookArea">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请填写您所在校区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_shangJia">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上架</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书籍版本信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>所在校区</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>微信</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QQ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>326</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请填写您的联系方式以便您不在线时买家联系您</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_weCHat">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>微信号中不能含有中文，否则上架失败！</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_QQ">
      <property name="placeholderText">
       <string>QQ号中不能含有中文，否则上架失败！</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pb_bookImg</zorder>
  <zorder>le_bookName</zorder>
  <zorder>le_bookInfo</zorder>
  <zorder>le_bookPrice</zorder>
  <zorder>le_bookArea</zorder>
  <zorder>pb_shangJia</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
